src/js/rs-release/rs-release.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * European Variation Archive (EVA) - Open-access database of all types of genetic
+ * variation data from all species
+ *
+ * Copyright 2020 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function EvaRsRelease(args) {
+    _.extend(this, Backbone.Events);
+    this.id = Utils.genId("EvaRsRelease");
+    _.extend(this, args);
+    this.render();
+}
+
+EvaRsRelease.prototype = {
+    render: function() {
+        var releaseVersion = $.urlParam('releaseVersion');
+        if (!releaseVersion) {
+            this.updateUrl({releaseVersion: 2})
```

Comment:
I am not a big fan of defaulting to a hardcoded release version... Could we make the API tolerant of absent releaseVersion parameter and return data for the most recent release version in such a scenario?

Reply:
I implemented an extra endpoint `/info/latest` to get the latest release and use it as the default if it is not specified in the url